frontend/libs/studio-feedback-form/src/FeedbackForm/Question/YesNoQuestion.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { StudioButton, StudioParagraph } from '@studio/components';
+import { ThumbDownFillIcon, ThumbDownIcon, ThumbUpFillIcon, ThumbUpIcon } from '@studio/icons';
+import type { QuestionsProps } from '../../types/QuestionsProps';
+
+type YesNoQuestionProps = QuestionsProps & {
+  buttonLabels: {
+    yes: string;
+    no: string;
+  };
+};
+
+export function YesNoQuestion({ id, label, value, buttonLabels, onChange }: YesNoQuestionProps) {
+  const setYes = () => {
+    onChange(id, 'yes');
+  };
+
+  const setNo = () => {
+    onChange(id, 'no');
+  };
+
+  const unsetYesOrNo = () => {
+    onChange(id, '');
+  };
+
+  return (
+    <div>
+      <StudioParagraph>{label}</StudioParagraph>
+      <div style={{ display: 'flex', flexDirection: 'row', gap: 12 }}>
+        {value !== 'yes' && (
+          <StudioButton
+            variant='tertiary'
+            size='lg'
+            icon={<ThumbUpIcon />}
+            onClick={setYes}
+            aria-label={buttonLabels.yes}
+          />
+        )}
+        {value === 'yes' && (
+          <StudioButton
+            variant='tertiary'
+            size='lg'
+            icon={<ThumbUpFillIcon />}
+            onClick={unsetYesOrNo}
+            aria-label={buttonLabels.yes}
+            aria-selected={true}
+          />
+        )}
```

Comment:
Would it simplify the code to use a single button and a ternary expression to dynamically adjust the props?

```suggestion
          <StudioButton
            variant='tertiary'
            size='lg'
            icon={<ThumbUpFillIcon />}
            onClick={value === 'yes' ? unsetYesOrNo : setYes}
            aria-label={buttonLabels.yes}
            aria-selected={value === 'yes'}
          />
        )}
```

Reply:
Yep! There are several props that are dependent on value, and I thought setting the buttons up explicitly was the clearest. But I tested your suggestion, with all of the affected props, and moving the checks into the component is actually much cleaner ❤️